Dockerfile

```diff
@@ -1,6 +1,20 @@
 # Dockerfile used in execution of Github Action
 FROM gruntwork/terragrunt:0.0.2
```

Comment:
I think should be update "FROM" image since was changed `terragrunt/Dockerfile`, if changes from `terragrunt/Dockerfile` will be published as `gruntwork/terragrunt:0.0.2` - it will break old versions

Reply:
ah good catch. Should it bump to `gruntwork/terragrunt:0.0.3`?

Reply:
I would say to 0.1.0 since it has a lot of changes